docs/onboard-vendors-to-techpass.md

```diff
@@ -36,8 +36,8 @@ Your project manager or reporting officer will work with the engaging government
 
 <details> <summary style="font-size:20px;font-weight:bold">Steps to sign in using TechPass username and initial password</summary>
 
-  1. 1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in with TechPass**.
-  <kbd>![log-in-with-techpass](assets/images/access-sgts-services-using-techpass/first.png)</kbd>
+  1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in with TechPass**.
```

Comment:
hmmm... do you want to provide the stg url as well?
so if staging users can use the right url for onboarding

https://stg.docs.developer.tech.gov.sg/